apis/core/lib/domain/cube-projects/update.ts

```diff
@@ -23,6 +23,7 @@ export async function updateProject({
   const project = await store.getResource<ImportProject | CsvProject>(resource.term)
 
   project.rename(resource.out(rdfs.label).value)
+  project.updateIsHiddenCube(resource.out(cc.isHiddenCube).term)
```

Comment:
And then set the flag

```suggestion
  project.isHiddenCube = $rdf.literal('true', xsd.boolean).equals(resource.out(cc.isHiddenCube).term))
```